src/index.tsx

```diff
@@ -125,8 +131,37 @@ async function activatePlugin(
 ): Promise<void> {
   const { commands } = app;
   const trans = translator.load('jupyterlab');
-  const fileBrowserTracker = browserFactory.tracker;
   const api = new SchedulerService({});
+
+  // Try calling an API to verify that the server extension is actually installed
+  let serverExtensionOk = false;
+  api
+    .getJobs({ max_items: 1 })
```

Comment:
Have we tried `max_items: 0`? This would slightly improve the initial extension load time.